.github/workflows/codeql-analysis.yml

```diff
@@ -37,10 +37,15 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@v3
+      
+    - name: Install dependencies
```

Comment:
I think there is something weird on your branch regarding the git history. These changes are already in `main`, and somehow they appear here as if you were bringing these new changes with your branch. 

Would it be possible for you to create you branch from scratch (from `main`), cherry-pick your commits and re-create the PR? I think this would make things easier to eventually merge your changes.

Otherwise it is difficult for the reviewers to know which are the real changes that you are brining.

Reply:
Hello @piponazo ,
ok I will do that ;
Thank you